conf.yaml

```diff
@@ -751,30 +761,35 @@ contents:
                   1.1: master
                   1.0: master
               -
-                alternatives: { source_lang: csharp, alternative_lang: go }
+                alternatives: { source_lang: console, alternative_lang: go }
                 repo:   clients-team
                 path:   examples/elastic-cloud/go
                 map_branches: *mapCloudEceToClientsTeam
               -
-                alternatives: { source_lang: csharp, alternative_lang: ruby }
+                alternatives: { source_lang: console, alternative_lang: ruby }
                 repo:   clients-team
                 path:   examples/elastic-cloud/ruby
                 map_branches: *mapCloudEceToClientsTeam
               -
-                alternatives: { source_lang: csharp, alternative_lang: java }
+                alternatives: { source_lang: console, alternative_lang: java }
                 repo:   clients-team
                 path:   examples/elastic-cloud/java
                 map_branches: *mapCloudEceToClientsTeam
               -
-                alternatives: { source_lang: csharp, alternative_lang: javascript }
+                alternatives: { source_lang: console, alternative_lang: javascript }
                 repo:   clients-team
                 path:   examples/elastic-cloud/javascript
                 map_branches: *mapCloudEceToClientsTeam
               -
-                alternatives: { source_lang: csharp, alternative_lang: python }
+                alternatives: { source_lang: console, alternative_lang: python }
                 repo:   clients-team
                 path:   examples/elastic-cloud/python
                 map_branches: *mapCloudEceToClientsTeam
+              -
+                alternatives: { source_lang: console, alternative_lang: csharp }
+                repo:   clients-team
+                path:   examples/elastic-cloud/csharp
+                map_branches: *mapCloudSaasToClientsTeam
```

Comment:
s/mapCloudSaasToClientsTeam/mapCloudEceToClientsTeam/

Reply:
Good catch! In my pre-meeting hurry to address Philip's comment, I messed this up. Fixed now.